community-meetings/2022-01-07.md

```diff
@@ -0,0 +1,18 @@
+# Agenda for the GoReleaser Community Meeting - January 7, 2022
+
+## Links for the participants
+
+- YouTube live stream /recording: [Youtube](https://www.youtube.com/channel/UCxg5N16FKrTa4Cees434pbw)
+- Current Milestones: [v1.1.0](https://github.com/goreleaser/goreleaser/milestone/3)
+
+## Topics
+
+- New Community Meetings Repo: [GoReleaser Community Meetings](https://goreleaser.com/community-meetings)
```

Comment:
This is a 404 for me.

Reply:
Good point, if @caarlos0 whants to bake this mds into the main page?

Reply:
I think the meetings repo is this one, isn't it? https://github.com/goreleaser/community